CONTRIBUTING.md

```diff
@@ -443,6 +443,15 @@ If you are contribuing visual assets from other open source repos, the source re
 
 1. Add an entry [here](docs/icons.md#third-party) summarizing the new destination location, where the assets were sourced from, and a brief rationale.
 
+### Custom Lint Rules
```

Comment:
let's put this above the `### AWS SDK generator`  section.

Reply:
any reason for putting it here?

Reply:
it's way more important than this other stuff near the bottom of the file